lms/djangoapps/verified_track_content/models.py

```diff
@@ -72,12 +75,32 @@ class VerifiedTrackCohortedCourse(models.Model):
         help_text=ugettext_lazy(u"The course key for the course we would like to be auto-cohorted.")
     )
 
+    verified_cohort_name = models.CharField(max_length=100, default=VERIFIED_COHORT_NAME)
```

Comment:
Also, I don't think it is realistic that course teams will wish to keep this name ("verified"). Here are the names that are currently being used--

"ID Verified Cohort"  (ASU)
"Micromasters" (MIT)

The name of the cohort will show in discussion posts-- "This post is visible only to verified."

How about we change the default to "Verified Learners"?

Of course my real concern here is that if course staff rename the cohort name in the instructor dashboard, they must also rename it here in the admin panel, else the functionality breaks.